events/button.js

```diff
@@ -56,7 +52,7 @@ module.exports = (client) => {
             interaction.deferUpdate()
         }
         if (interaction.customId.startsWith("gwjoin")) {
-            let gameName = interaction.customId.split("-").slice(1).join("-")
+            let gameName = interaction.customId.split("-")[1]
```

Comment:
allowing "-" in the game name 
```suggestion
            let gameName = interaction.customId.split("-").slice(1).join("-")
```